### ISSUE TITLE
Field class names get appended to instead of overwritten when outputting multiple forms on the same page

### ISSUE THREAD

**Issue by thisisjamessmith** [solspace/craft3-freeform#190]
**Describe the bug or issue you're experiencing**

I'm trying to output the same form twice on a single page. When calling `field.renderInput()`, the input's class attribute  on the second form is populated with the class from the call in the first form and the second form concatenated together.

**Steps to reproduce**

Here's a reduced test case:

```
{% set form = craft.freeform.form('newsletterSignup') %}

{{ form.renderTag({
	id:'form1',
	fieldIdPrefix: 'form1-'
}) }}
	{% for row in form %}
		{% for field in row %}
			{{ field.renderInput({ class: 'aaa' }) }}
		{% endfor %}
	{% endfor %}
{{ form.renderClosingTag }}

<hr><hr><hr>

{{ form.renderTag({
	id:'form2',
	fieldIdPrefix: 'form2-'
}) }}
	{% for row in form %}
		{% for field in row %}
			{{ field.renderInput({ class: 'bbb' }) }}
		{% endfor %}
	{% endfor %}
{{ form.renderClosingTag }}
```

**Expected behavior**

The input element that is rendered by the second form has `class="aaa bbb"` when it should just have `class="bbb"`.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.6.11.2
- Freeform Version: 3.11.4
- Freeform Edition: Lite

**Comment by kjmartens** [comment]
Hi @thisisjamessmith,

Sorry for the trouble here. I can duplicate this on my end as well. It's currently something I don't think we can correct in Freeform 3.11.x, but we're working on and close to beginning the 3.12 beta, which makes some drastic improvements and refactoring to the way these things behave. We'll be able to more easily correct the issue for 3.12. It could be about 1-2 weeks away from this issue being addressed.

I'm very sorry for the inconvenience here.

**Comment by kjmartens** [comment]
@thisisjamessmith I think appears to be fixed in 3.12 now. Feel free to try it out. :)

**Comment by thisisjamessmith** [comment]
Thanks, I've just tested this same test case above on the latest Freeform, and it's still not working.
